MinStack.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Stack;
+
+// Time Complexity : O(1)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : Did not get the logic to solve isFull() 
```

Comment:
Are we using isFull() anywhere?

Reply:
No, but as it was asked the problem statement that is why I was wondering "Design a Data Structure SpecialStack that supports all the stack operations like push(), pop(), isEmpty(), isFull()"